src/zope/interface/tests/test_verify.py

```diff
@@ -15,12 +15,29 @@
 """
 import unittest
 
+# pylint:disable=inherit-non-class,no-method-argument,no-self-argument
 
 class Test_verifyClass(unittest.TestCase):
 
-    def _callFUT(self, iface, klass):
+    verifier = None
+
+    @classmethod
+    def setUpClass(cls):
+        # zope.testrunner doesn't call setUpClass, so if you get
+        # 'NoneType is not callable', that's why.
+        cls.verifier = staticmethod(cls._get_FUT())
+
+    @classmethod
+    def _get_FUT(cls):
         from zope.interface.verify import verifyClass
-        return verifyClass(iface, klass)
+        return verifyClass
+
+    _adjust_object_before_verify = lambda self, x: x
+
+    def _callFUT(self, iface, klass, **kwargs):
+        return self.verifier(iface,
+                             self._adjust_object_before_verify(klass),
+                             **kwargs)
```

Comment:
Ok, this is funky sleight of hand I don't understand, but maybe things will become clearer if I read more of the diff?

Reply:
(no I did not understand why all this magic was necessary)

Reply:
Which part seems magic? 

The idea here was that rather than redefining the entirety of `_callFUT` in the subclass (I tried a few variations on that, all of them were repetitive), the subclass just defines `adjust_object_before_verify`.